package.nix

```diff
@@ -7,6 +7,5 @@ pkgs.buildGo124Module rec {
   version = "${builtins.hashFile "sha256" "${toString ./go.sum}"}";
   src = ./.;
   vendorHash = "sha256-IxjJNjF+fp4iLmo/MW5nJOx7HKuFhw7zgWqg/0remhk=";
-  subPackages = ["cmd/paretosecurity"];
```

Comment:
`subPackages` has a bug that if it is set, then only tests for `cmd/paretosecurity` are run, which effectively means no tests are run. There's a draft PR to fix this, but it's not gonna happen soon.